airflow/contrib/operators/aws_athena_operator.py

```diff
@@ -74,7 +78,17 @@ def execute(self, context):
         self.result_configuration['OutputLocation'] = self.output_location
         self.query_execution_id = self.hook.run_query(self.query, self.query_execution_context,
                                                       self.result_configuration, self.client_request_token)
-        self.hook.poll_query_status(self.query_execution_id)
+        query_status = self.hook.poll_query_status(self.query_execution_id, self.max_tries)
+
+        if query_status in AWSAthenaHook.FAILURE_STATES:
+            raise Exception(
+                'Final state of Athena job is {}, query_execution_id is {}.'
+                .format(query_status, self.query_execution_id))
+        elif not query_status or query_status in AWSAthenaHook.INTERMEDIATE_STATES:
+            raise Exception(
+                'Final state of Athena job is {}. \
+                 Max tries of poll status exceeded, query_execution_id is {}.'
```

Comment:
Hi @bryanyang0528 This line-breaking here will cause minor display issue. 

You can try to run
```python
raise Exception(
    'Final state of Athena job is {}. \
    Max tries of poll status exceeded, query_execution_id is {}.'
    .format("AAA", "BBB"))
```

What you will see is 
```
Exception: Final state of Athena job is AAA.     Max tries of poll status exceeded, query_execution_id is **BBB.**
```
in which we have unnecessary extra spaces between the two sentences.

Please change how you break the line. Thanks.

Reply:
@XD-DENG Thanks for reviewing.

Reply:
Kindly allow me a bit more time to have a closer look at your test update before I merge.

Reply:
No problem. Thx